### ISSUE TITLE
Questions about training loss

### ISSUE THREAD

**Issue by SmashChen** [autonomousvision/transfuser#164]
Dear Authors,
  Thank you for your great work! In PAMI22,  could you offer me the approximate loss in training and validation for waypoints and other auxiliary tasks?  Thank you very much!

**Comment by Kait0** [comment]
We didn't use validation losses for the experiments in the paper as we trained on all the data.
Training losses were somethink like:
Loss_bev: 0.13
loss_center_heatmap: 0.03
loss_depth: 0.199
loss_offset: 0.02
loss_seg: 0.05
loss_total: 0.99
loss_wh: 0.01
loss_wp: 0.04
loss_yaw_class: 0.02
loss_yaw_res: 0.0005

Open loop losses are not really indicative of closed loop performance, I typically don't pay much attention to them.
But you can see whether there is some optimization problem, losses should always go down.
Should look something like this:

![image](https://github.com/autonomousvision/transfuser/assets/4252687/c80086ea-afb4-4662-8a78-d27847e0671c)

**Comment by SmashChen** [comment]
Thank you very much for your reply!
